<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>843</height>
   </rect>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
                    background-color: rgb(255, 164, 134);
                    background-color: rgb(88, 88, 88);
                </string>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">
                                color: rgb(255, 255, 255);
                                selection-background-color: #4CAF50;
                            </string>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>147</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>64</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
        </font>
       </property>
      </column>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout">
      <property name="horizontalSpacing">
       <number>7</number>
      </property>
      <property name="verticalSpacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Название:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="materialLineEdit">
        <property name="styleSheet">
         <string>
                                        QLineEdit {
                                        background-color: #f0f0f0;
                                        border: 1px solid #ccc;
                                        border-radius: 5px;
                                        padding: 8px;
                                        font-size: 14px;
                                        }
                                    </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Тип работы:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="workTypeLineEdit">
        <property name="styleSheet">
         <string notr="true">
                                        QLineEdit {
                                        background-color: #f0f0f0;
                                        border: 1px solid #ccc;
                                        border-radius: 5px;
                                        padding: 8px;
                                        font-size: 14px;
                                        }
                                    </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Автор:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="authorLineEdit">
        <property name="styleSheet">
         <string notr="true">
                                        QLineEdit {
                                        background-color: #f0f0f0;
                                        border: 1px solid #ccc;
                                        border-radius: 5px;
                                        padding: 8px;
                                        font-size: 14px;
                                        }
                                    </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Группа:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="groupLineEdit">
        <property name="styleSheet">
         <string>
                                        QLineEdit {
                                        background-color: #f0f0f0;
                                        border: 1px solid #ccc;
                                        border-radius: 5px;
                                        padding: 8px;
                                        font-size: 14px;
                                        }
                                    </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Дисциплина:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="disciplineLineEdit">
        <property name="styleSheet">
         <string notr="true">
                                        QLineEdit {
                                        background-color: #f0f0f0;
                                        border: 1px solid #ccc;
                                        border-radius: 5px;
                                        padding: 8px;
                                        font-size: 14px;
                                        }
                                    </string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Год публикации:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="yearLineEdit">
        <property name="styleSheet">
         <string notr="true">
                                        QLineEdit {
                                        background-color: #f0f0f0;
                                        border: 1px solid #ccc;
                                        border-radius: 5px;
                                        padding: 8px;
                                        font-size: 14px;
                                        }
                                    </string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
        <property name="cursorMoveStyle">
         <enum>Qt::LogicalMoveStyle</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Место хранения:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="storageLineEdit">
        <property name="styleSheet">
         <string notr="true">
                                        QLineEdit {
                                        background-color: #f0f0f0;
                                        border: 1px solid #ccc;
                                        border-radius: 5px;
                                        padding: 8px;
                                        font-size: 14px;
                                        }
                                    </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string>
                                        QPushButton {
                                        background-color: #2da9d2;
                                        color: white;
                                        border-radius: 5px;
                                        font-size: 16px;
                                        padding: 8px 16px;
                                        }

                                        QPushButton:hover {
                                        background-color: #258eb1;
                                        }

                                        QPushButton:pressed {
                                        background-color: #1b6a85;
                                        }
                                    </string>
        </property>
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>18</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
                                        QPushButton {
                                        background-color: #4CAF50;
                                        color: white;
                                        border-radius: 5px;
                                        font-size: 16px;
                                        padding: 8px 16px;
                                        }
                                        QPushButton:hover {
                                        background-color: #45a049;
                                        }
                                        QPushButton:pressed {
                                        background-color: #37823b;
                                        color: white;
                                        border: 1px solid #37823b;
                                        }
                                    </string>
        </property>
        <property name="text">
         <string>Добавить работу</string>
        </property>
       </widget>
      </item>
         <item>
    <widget class="QPushButton" name="editButton">
        <property name="font">
            <font>
                <family>MS Shell Dlg 2</family>
                <pointsize>-1</pointsize>
                <weight>75</weight>
                <italic>false</italic>
                <bold>true</bold>
            </font>
        </property>
        <property name="styleSheet">
            <string notr="true">
                QPushButton {
                    background-color: #ffb100; /* цвет подходящий для редактирования */
                    color: white;
                    border-radius: 5px;
                    font-size: 16px;
                    padding: 8px 16px;
                }
                QPushButton:hover {
                    background-color: #e69e00; /* цвет при наведении */
                }
                QPushButton:pressed {
                    background-color: #cc8400; /* цвет при нажатии */
                    color: white;
                    border: 1px solid #cc8400;
                }
            </string>
        </property>
        <property name="text">
            <string>Редактировать</string>
        </property>
    </widget>
</item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
                                        QPushButton {
                                        background-color: red;
                                        color: white;
                                        border-radius: 5px;
                                        font-size: 16px;
                                        padding: 8px 16px;
                                        }
                                        QPushButton:hover {
                                        background-color: #a50000;
                                        color: white;
                                        border: 1px solid #a50000;
                                        }
                                        QPushButton:pressed {
                                        background-color: #610a0a;
                                        color: white;
                                        border: 1px solid #610a0a;
                                        }
                                    </string>
        </property>
        <property name="text">
         <string>Удалить работу</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
